docs/reference-implementation/Getting-Started.md

```diff
@@ -20,7 +20,7 @@ The following tools and applications must be installed on the client machine use
 
 - Install [Azure DevOps CLI](https://docs.microsoft.com/azure/devops/cli/?view=azure-devops)
```

Comment:
I would mark this as optional - based on whether somebody wants to follow the CLI path or not